src/styles.scss

```diff
@@ -31,3 +31,7 @@ a {
   position: relative;
   z-index: 997;
 }
+
+.metrics-icon {
+  color: #64b5f6;
```

Comment:
This should be in metrics.component.scss (this is not a global style) and shouldn't be hardcoding the color, you should use the color form styles-colors.scss